src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,191 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node head;
+    private Node tail;
+
     @Override
     public void add(T value) {
+        if (size == 0) {
+            addIfSize0(value);
```

Comment:
```suggestion
            addHead(value);
```
if the size is 0, you add a node, set it both as a head and tail, you can add a second node just as any other